pkg/repository/udmrepo/kopialib/lib_repo.go

```diff
@@ -79,6 +79,8 @@ const (
 	defaultMaintainCheckPeriod     = time.Hour
 	overwriteFullMaintainInterval  = time.Duration(0)
 	overwriteQuickMaintainInterval = time.Duration(0)
+	eagerGCFullInterval            = time.Duration(12)
+	eagerGCQuickInterval           = time.Duration(6)
```

Comment:
I think these two params aren't used anymore, since fast and eager full GC intervals are defined elsewhere now.